### ISSUE TITLE
A11y: Icons should be marked up with 'aria-hidden'

### ISSUE THREAD

**Issue by timo-haas-gh** [shopware/platform#7369]
*PHP Version:* 8.3   | *Shopware Version:* 6.6.10.2  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

Icons are currently not marked up with aria-hidden.

![Image](https://github.com/user-attachments/assets/e866d89a-4b15-4a4b-a873-5cf73d572d50)

*_Expected behaviour:_*

Icons should be marked up with aria-hidden if they have a superior marked up node.

*_How to reproduce:_*

Check the storefront with ACCESSIBILITY_TWEAKS

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-41030**



[created from NEXT-41030, comment 543638]

**Comment by tobiasberge** [comment]
Hi @timo-haas-gh,

thank you for helping us to improve accessibility. We have implemented a change to make `sw_icon` icons `aria-hidden` by default. It was also set to ariaHidden manually in many places already. And like you pointed out, the label should mostly be on a parent node with hidden label or aria-label.
